### ISSUE TITLE
Can easily be uninstallable even locking playstore and settings

### ISSUE THREAD

**Issue by hemanta212** [lizixian18/temporary#9]
I have locked playstore and settings but still I can easily uninstall this app in launcher.
Android version: 4.4.2

**Comment by alex-theoto** [comment]
Yes, there must be another application on the blacklist to block the uninstall procedure if you long press the app and drag it to 'uninstall' option.
This happens even on the blacklisted apps.

Maybe the 'package installer' application from the system apps.
